src/App/App.tsx

```diff
@@ -47,15 +47,25 @@ export default function App() {
               <Route path={`${appRoutes.donate}/:id`} element={<Donate />} />
               <Route path={appRoutes.leaderboard} element={<Leaderboard />} />
               <Route
+                index={IS_AST}
```

Comment:
There's no need to set `Route.index` either for "Register" nor for "Marketplace".
- "Registration" & "Launchpad" pages can be always set to `${appRoutes.register}/*` path
- "Marketplace" page can be always set to `appRoutes.marketplace` path
- `Navigate.to` prop should be updated like `to={IS_AST ? appRoutes.register : appRoutes.marketplace}`

This would also solve both https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2056/files#r1163778332 and https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2056/files#r1163789200